src/main/java/entities/post.java

```diff
@@ -0,0 +1,118 @@
+package entities;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * The Post class contains all the necessary information of a post: its ID, title,
+ * description,tags,collaborators, deadline, and users who favourited it.
+ */
+public class Post extends Postable{
+    public static int numPostsCreated = 0;
+    /**The id is equal to the number of posts created.*/
+    private int id;
+    private Timer timer;
+    /**This is a list of the users who favourited this post.*/
+    private List<User> favouritedUsers;
+    private String title;
+    private List<String> tags;
+    private String collaborators;
+    private Date deadline;
+
+    /**
+     * @param poster the User who is posting this Post. This is an attribute of the superclass Postable.
+     * @param timer an object that keeps track of time and deletes the Post when the deadline arrives.
+     * @param title this Post's title.
+     * @param mainDesc this Post's description.
+     * @param tags the tags this Post is added to.
+     * @param collaborators a description of the type of the collaborators the User who posted this Posts is looking for.
+     * @param deadline the date on which the Post is to be deleted from the database.
+     * @param replies the comments that are added to this post. This is an attribute of the superclass Postable.
+     * @param numPostsCreated1 the number of posts created so far.
```

Comment:
Pedantic I know, but small typo in Line 30 numPostsCreated

Reply:
Because numPostsCreated is a static variable, I thought it would be better if I make the parameter a different name than numPostsCreated since I cannot use "this." with a static var.